addons/blocked_locations/migrations/0.2/pre-migrate.py

```diff
@@ -0,0 +1,3 @@
+def migrate(cr, version):
+    # Update blocked reasons with dummy reasons since we're making this field a required field if the location is blocked.
+    cr.execute("UPDATE stock_location SET u_blocked_reason = 'Location is blocked' WHERE u_blocked IS True and u_blocked_reason IS Null;")
```

Comment:
nit: format this more like:
```python
    cr.execute("""
        UPDATE stock_location
        SET u_blocked_reason = 'Location is blocked'
        WHERE u_blocked IS True and u_blocked_reason IS Null;
    """) 
```

Reply:
and a new line =) to get rid of the angry red minus sign =D